test/utils.test.js

```diff
@@ -18,6 +18,23 @@ test('Build Workspace Endpoint URL', () => {
     expect(wayscriptClient.buildURLEndpoint("workspaces","detail",{"id":id})).toBe("https://api.wayscript.com/workspaces/d1e498e4-2f32-4e5c-803e-d5fe1e2b89fe");
 });
 
+test('Get Environment Variables', () => {
+    let env_vars = [
+        "WAYSCRIPT_EXECUTION_USER_TOKEN",
+        "WAYSCRIPT_EXECUTION_USER_REFRESH_TOKEN",
+        "WAYSCRIPT_EXECUTION_USER_APPLICATION_KEY",
+        "WAYSCRIPT_LAIR_URL",
+        "WS_PROCESS_ID"
+    ];
+    
+    env_vars.forEach(v => process.env[v] = "TEST_SETTING");
```

Comment:
I highly recommend setting each of these to different values to ensure that the correct env var is read.  I notice we do something similar in python and I don't feel great about it.

Reply:
Done.